pkgs/development/python-modules/torchvision/default.nix

```diff
@@ -1,30 +1,41 @@
-{ lib
-, symlinkJoin
-, buildPythonPackage
+{ buildPythonPackage
+, cudaSupport ? torch.cudaSupport or false # by default uses the value from torch
 , fetchFromGitHub
-, ninja
-, which
+, lib
 , libjpeg_turbo
 , libpng
+, ninja
 , numpy
-, scipy
 , pillow
-, torch
 , pytest
-, cudaSupport ? torch.cudaSupport or false # by default uses the value from torch
+, scipy
+, symlinkJoin
+, torch
+, which
 }:
 
 let
-  inherit (torch) gpuTargetString;
-  inherit (torch.cudaPackages) cudatoolkit cudnn;
+  inherit (torch) cudaPackages gpuTargetString;
```

Comment:
Outside the scope of PR, but maybe we could move this to the argument list, kind of:
```diff
, which
- }:
+ gpuTargetString ? torch.gpuTargetString
+ }:

let
-  inherit (torch) cudaPackages gpuTargetString;
+  inherit (torch) cudaPackages;
```

This came up just now during testing: I wanted to re-build `torchvision` without rebuilding `torch`

Reply:
From a brief look at the updated cuda section in the manual, it would be logical to take `cudaCapabilities` as an argument, rather than the compiled string. And to expose that in `python3Packages.torch` via `passthru`

...which gets even further beyond the scope, so let's skip it

Reply:
See https://github.com/NixOS/nixpkgs/issues/220377